src/main.cpp

```diff
@@ -826,116 +826,63 @@ void setup()
 #endif
 
 #ifdef ARCH_PORTDUINO
-    if (settingsMap[use_sx1262]) {
-        if (!rIf) {
-            LOG_DEBUG("Activate sx1262 radio on SPI port %s", settingsStrings[spidev].c_str());
-            if (settingsStrings[spidev] == "ch341") {
+    const struct { configNames cfgName;
+                   std::string strName;
+                   bool ch341halsupport;
```

Comment:
There's nothing inherently preventing the other chips from having ch341 hal support, just not any available hardware yet. That part can be removed to simplify this code a bit.

Reply:
Fixed this part. I think in the long run, the CH341 SPI check needs to reworked anyway. What it truly tries to do: check for an user-space driver, which in this case is CH341. I am playing currently a bit with the [MCP2210](https://www.microchip.com/en-us/product/mcp2210), which is a bit more expensive but more adjustable. However, in case on wants to use the MCP2210 in the userspace, the CH341 flag won't be functioning that well anymore.